force-app/main/default/objectTranslations/Bucket__mdt-de/Bucket__mdt-de.objectTranslation-meta.xml

```diff
@@ -1,44 +1,51 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <layouts>
+        <layout>Bucket Layout</layout>
+        <sections>
+            <label>Benutzerdefinierte Links</label>
+            <section>Custom Links</section>
+        </sections>
+    </layouts>
+    <gender>Masculine</gender>
     <caseValues>
-        <caseType>Nominative</caseType>
         <plural>false</plural>
-        <value><!-- Bucket --></value>
-    </caseValues>
-    <caseValues>
         <caseType>Nominative</caseType>
-        <plural>true</plural>
-        <value><!-- Bucket --></value>
+        <value>Bucket</value>
     </caseValues>
     <caseValues>
-        <caseType>Accusative</caseType>
         <plural>false</plural>
-        <value><!-- Bucket --></value>
-    </caseValues>
-    <caseValues>
         <caseType>Accusative</caseType>
-        <plural>true</plural>
-        <value><!-- Bucket --></value>
+        <value>Bucket</value>
     </caseValues>
     <caseValues>
-        <caseType>Genitive</caseType>
         <plural>false</plural>
-        <value><!-- Bucket --></value>
+        <caseType>Dative</caseType>
+        <value>Bucket</value>
     </caseValues>
     <caseValues>
+        <plural>false</plural>
         <caseType>Genitive</caseType>
+        <value>Buckets</value>
+    </caseValues>
+    <caseValues>
         <plural>true</plural>
-        <value><!-- Bucket --></value>
+        <caseType>Nominative</caseType>
+        <value>Buckets</value>
     </caseValues>
     <caseValues>
-        <caseType>Dative</caseType>
-        <plural>false</plural>
-        <value><!-- Bucket --></value>
+        <plural>true</plural>
+        <caseType>Accusative</caseType>
+        <value>Buckets</value>
     </caseValues>
     <caseValues>
+        <plural>true</plural>
         <caseType>Dative</caseType>
+        <value>Buckets</value>
+    </caseValues>
+    <caseValues>
         <plural>true</plural>
-        <value><!-- Bucket --></value>
+        <caseType>Genitive</caseType>
+        <value>Buckets</value>
```

Comment:
@AndersonTarren - It seems like german wasn't fully translated here? custom metadata translations have issues and aren't fully functional but thinking we should still have these translated for when it is ready.